docs/concepts/autoscaling.md

```diff
@@ -3,3 +3,19 @@ sidebar_position: 4
 ---
 
 # Autoscaling
+
+## Overview
+
+Responsive automatically manages your applications to meet changing workload demands. You simply define a policy that specifies health goals for the application. The Responsive Controller combines the policy with metrics and status updates from the application and Responsive Operator to decide how to keep your cluster healthy. Then, it coordinates with the application and Responsive Operator to automatically execute remediating actions.
```

Comment:
super nit, but can we newline at ~80 chars?

Reply:
yep. easy enough with vim